packages/vite/src/streamHelpers.ts

```diff
@@ -0,0 +1,146 @@
+import { Writable } from 'node:stream'
+import path from 'path'
```

Comment:
Nit for consistency between the imports here:

```suggestion
import { Writable } from 'node:stream'
import path from 'node:path'
```